zqd/search/zng.go

```diff
@@ -69,3 +69,7 @@ func (r *ZngOutput) SendControl(ctrl interface{}) error {
 	r.flush()
 	return nil
 }
+
+func (r *ZngOutput) ContentType() string {
+	return "application/x-zng"
```

Comment:
This should be a `const` in api so the client also has access to this and can respond appropriately.